css/index.css

```diff
@@ -1,3 +1,405 @@
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+/* Services Styles */
+
+.services-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    line-height: 1.3;
+}
+
+@media (max-width: 800px) {
+    .services-content {
+        display: flex;
+        flex-wrap: wrap;
+        width: 90%;
+        margin: 0;
+    }
+}
+
+.services-content .services-header-img {
+    margin: 20px 0;
+}
+
+@media (max-width: 800px) {
+    .services-content .services-header-img {
+        width: 90%;
+    }
+}
+
+@media (max-width: 500px) {
+    .services-content .services-header-img {
+        width: 90%;
+    }
+}
+
+.services-content h1 {
+    font-size: 26px;
+    margin: 10px 0;
+}
+
+@media (max-width: 500px){
+    .services-content {
+        display: flex;
+        flex-direction: column;
+        margin: 0 auto;
+    }
+}
+
+
+.services-content .services-text {
+ border-bottom: 1px solid black;
+ padding-bottom: 40px;
+ margin-bottom: 20px
+}
+
+@media (max-width: 800px) {
+    .services-content .services-text {
+        width: 90%;
+        height: auto;
+    }
+}
+
+@media (max-width: 500px) {
+    .services-content .services-text {
+        width: 45%;
+        height: auto;
+        margin-right: 5%;
+    }
+}
+
+
+
+.services-content .card {
+    padding: 20px;
+    border: 1px solid black;
+    width: 420px;
+    background: whitesmoke;
+    margin: 20px 0;
+}
+
+.services-content .card button {
+    padding: 10px 40px;
+    margin-top: 20px;
+    cursor: pointer;
+    background: white;
+    border-radius: 5px;
+    border: 1px solid black;
+}
+
+.services-content .card button:hover {
+    background: black;
+    color: white;
+} 
+
+@media (max-width: 800px) {
+    .services-content .card {
+        width: 40%;
+        margin: 10 0px;
+    }
+}
+
+@media (max-width: 500px) {
+    .services-content .card {
+        height: 80%;
+        padding: 2%;
+        letter-spacing: 1;
+    }
+}
+
+.services-content .services-info ul {
+    margin: 20px;
+}
+
+.services-content .services-info ul li{
+    list-style: square;
+    padding: 5px;
+}
+
+.services-content .services-info {
+    display: flex;
+    flex-wrap: wrap;
+    border-top: 2px solid black; 
+    margin: 20px 0 35px 0;
+    padding-top: 40px; 
+}
+
+@media (max-width: 800px) {
+    .services-content .services-info {
+        width: 90%;
+        height: auto;
+    }
+}
+
+@media (max-width: 500px) {
+    .services-content .services-info {
+        width: 45%;
+        height: auto;
+        padding-top: 5%;
+      }
+}
+
+.services-content .services-info .facts {
+  width: 400px;
+  margin: 0 60px 20px 0;
+}
+
+.services-content .services-info .facts p {
+  margin-bottom: 20px;
+}
+
+.services-content .services-info .services-info-img {
+  width: 420px;
+  height: 300px;
+}
+
```

Comment:
Sometimes it is easier to read if all the CSS selectors are included in the same media query. Also, people sometimes put them at the bottom to cascade correctly. The ones at the bottom will overwrite the ones at the top.